packages/cli/src/lib/defaults/infra-modules/http/server/src/utils/uuid.ts

```diff
@@ -0,0 +1,11 @@
+export function uuidV4() {
```

Comment:
Why not use a uuid library for this? We do this here: https://github.com/polywrap/toolchain/blob/0c12092d8609c2254060f626f2219f2271b68b78/packages/js/client/src/PolywrapClient.ts#L1